db/migrate/20171223044916_create_pghero_query_stats_old.rb

```diff
@@ -1,4 +1,4 @@
-class CreatePgheroQueryStats < ActiveRecord::Migration[5.1]
+class CreatePgheroQueryStatsOld < ActiveRecord::Migration[5.1]
```

Comment:
I feel sort of bad changing these file names and class names, but otherwise Rails was complaining about the migrations already existing.